### PR TITLE
Add multiple network cal procedure

### PR DESCRIPTION
**Pull request by shanshanhsieh**
This pull request modifies the `thermal_network_matrix.py` to iterate through multiple networks within a district.

To test:
- sync 'cea-reference-case'
- there should be four folders (All_building, RTP, SBS, N3_1) under this directory `\reference-case-ecocampus\baseline\inputs\networks\DC'
- run 'thermal_network_matrix.py' with the scenario 'reference-case-ecocampus`
- the script should execute the DC network calculation for all four networks

### CODE REVIEW

**Comment by daren-thomas** [comment]
Build finished.

File: cea/technologies/thermal_network/thermal_network_matrix.py [hunk 1]
```diff
@@ -103,10 +104,16 @@ def thermal_network_main(locator, gv, network_type, source, set_diameter):
 
     # # prepare data for calculation
 
-    # read building names
+    # read building names from the entire district
     total_demand = pd.read_csv(locator.get_total_demand())
     building_names = total_demand['Name']
```
Comment (BhargavaKrishnaSreepathi) [inline cea/technologies/thermal_network/thermal_network_matrix.py]: this line can be replaced by `building_names = pd.read_csv(locator.get_total_demand())['Name'].values`. This prevents loading the entire dataframe

Comment (shanshanhsieh) [reply cea/technologies/thermal_network/thermal_network_matrix.py]: done!

**Comment by daren-thomas** [comment]
Build finished.

**Comment by BhargavaKrishnaSreepathi** [comment]
good work @shanshan

**Pull request merged**